index.js

```diff
@@ -12,20 +12,21 @@ function formatPlugins(listOfPlugins) {
 }
 
 async function buildBundle(bundleConfig, pluginsFromConfig = []) {
-    // only transpile not in development?
     const inputOpts = {
         input: path.resolve(bundleConfig.entry),
         plugins: formatPlugins(pluginsFromConfig)
     };
 
-    const outputOpts = {
-        file: path.resolve(bundleConfig.dest),
-        format: bundleConfig.format || 'es'
-    };
-
     try {
         const bundle = await rollup.rollup(inputOpts);
-        return bundle.write(outputOpts);
+        const outputConfigs = Array.isArray(bundleConfig.output) ? bundleConfig.output : [bundleConfig.output];
+        return Promise.all(outputConfigs.map(oConfig => {
+            const outputOpts = {
+                file: path.resolve(oConfig.dest),
+                format: oConfig.format || 'es'
```

Comment:
IIFE bundles require (I think) an additional output property: `name`.  This is for assigning the IIFE to a global variable. And there are actually quite a few output properties available for rollup: https://rollupjs.org/guide/en#javascript-api.  Could the config.output objects contain any number of properties that get passed through to rollup? That way we're allow the user of this plugin to take full advantage of Rollup's features.